README.md

```diff
@@ -42,14 +42,29 @@ Then run:
 ```
 docker build -t rodekruis510/typhoonibf .
 ```
+Create a docker container
+```
+docker run --name typhoonibf rodekruis510/typhoonibf
 
+```
+Run and access the container
+```
+docker run -it --entrypoint /bin/bash rodekruis510/typhoonibf
```

Comment:
This command is equivalent to the one on line 57, except it's missing the volume mounting component. This might be why you're finding that the /home/fbf directory is empty